pkg/analysis/builtins.go

```diff
@@ -154,6 +154,12 @@ func LoadBuiltinsFromSource(ctx context.Context, contents []byte, path string) (
 
 	doc.Close()
 
+	// NewDocument returns these symbols with a location of __init__.py, which isn't helpful to anyone
```

Comment:
Yeah, I was thinking the same that the builtins wouldn't have a definition to go to. Unless we extracted the builtins *.py to the filesystem and popped that open in the editor. Still don't think that would be useful since it was just docs.

Reply:
I'd actually rather like to be able to link to the docs, since the tooltips aren't very readable. I tried just returning the location as https://docs.tilt.dev/api.html#func and VSCode didn't like that.